src/main/java/org/testng/GuiceHelper.java

```diff
@@ -41,24 +43,26 @@ Injector getInjector(IClass iClass, IInjectorFactory injectorFactory) {
     }
     Injector parentInjector = ((ClassImpl) iClass).getParentInjector(injectorFactory);
 
-    List<Module> moduleInstances =
-        Lists.newArrayList(getModules(guice, parentInjector, iClass.getRealClass()));
+    List<Module> classLevelModules = getModules(guice, parentInjector, iClass.getRealClass());
```

Comment:
Is removing the `Lists.newArrayList` safe enough?

Reply:
Yes. It is safe since `getModules` takes up that responsibility of creating a new list